shared/common-adapters/dumb.desktop.js

```diff
@@ -40,6 +41,20 @@ const checkboxMap: DumbComponentMap<Checkbox> = {
   },
 }
 
+const iconMap: DumbComponentMap<Icon> = {
```

Comment:
We just saw this break. Also with vertical metrics and etc who knows. If we
need to aggregate this into a single sheet we can

On Wednesday, July 6, 2016, Max Goodman <notifications@github.com> wrote:

> In shared/common-adapters/dumb.desktop.js
> <https://github.com/keybase/client/pull/3351#discussion_r69828995>:
>
> > @@ -40,6 +41,20 @@ const checkboxMap: DumbComponentMap<Checkbox> = {
> >    },
> >  }
> >
> > +const iconMap: DumbComponentMap<Icon> = {
>
> This will probably be pretty slow -- to avoid timing issues we're
> currently waiting 250ms per dumb map item. Can experiment with reducing the
> sleep. On the whole though, I don't see why this is necessary -- shouldn't
> the icons render the same each time? Won't this give the same exact results
> as the git file diff of the icon files?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3351/files/7048276b85b76ffd51cdb018705ac566835b1006#r69828995>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhU0osySMqQkFoxuL36Oc6dbJc_EAGks5qTDf_gaJpZM4JGL0L>
> .
>